contracts/cw20-escrow/src/state.rs

```diff
@@ -77,7 +77,7 @@ impl Escrow {
         }
 
         if let Some(end_time) = self.end_time {
-            if env.block.time > end_time {
+            if env.block.time > Timestamp::from_seconds(end_time) {
```

Comment:
You could turn the state into `pub end_time: Option<Timestamp>` now if you want

Reply:
I rather keep the user-facing API in seconds.
I guess this is state, but then we expose this in query.